src/main/scala/io/citrine/lolo/Model.scala

```diff
@@ -2,15 +2,15 @@ package io.citrine.lolo
 
 import breeze.linalg.DenseMatrix
 
-trait Model[+T <: PredictionResult[Any]] extends Serializable {
+trait Model[+T] extends Serializable {
```

Comment:
This is a good change

Reply:
Agreed. The generic now always means "the type of label data" across the package. I don't know why this ever deviated from that.